lib/util/simulcast.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
```

Comment:
We are only adding this file for use in the tests, right? Can we move it to `test/` then?